src/main/scala/chisel3/simulator/package.scala

```diff
@@ -107,25 +107,38 @@ package object simulator {
         )
       )
 
-      // Move the relevant files over to primary-sources
-      val filelist =
-        new java.io.BufferedReader(new java.io.FileReader(s"${workspace.supportArtifactsPath}/filelist.f"))
-      try {
-        filelist.lines().forEach { immutableFilename =>
-          var filename = immutableFilename
-          /// Some files are provided as absolute paths
-          if (filename.startsWith(workspace.supportArtifactsPath)) {
-            filename = filename.substring(workspace.supportArtifactsPath.length + 1)
+      // Move the files indicated by a filelist.
+      def moveFiles(filename: String) = {
+        val filelist = new java.io.BufferedReader(new java.io.FileReader(filename))
+        try {
+          filelist.lines().forEach { immutableFilename =>
+            var filename = immutableFilename
+            /// Some files are provided as absolute paths
+            if (filename.startsWith(workspace.supportArtifactsPath)) {
+              filename = filename.substring(workspace.supportArtifactsPath.length + 1)
+            }
+            java.nio.file.Files.move(
+              java.nio.file.Paths.get(s"${workspace.supportArtifactsPath}/$filename"),
+              java.nio.file.Paths.get(s"${workspace.primarySourcesPath}/$filename")
+            )
           }
-          java.nio.file.Files.move(
-            java.nio.file.Paths.get(s"${workspace.supportArtifactsPath}/$filename"),
-            java.nio.file.Paths.get(s"${workspace.primarySourcesPath}/$filename")
-          )
+        } finally {
+          filelist.close()
         }
-      } finally {
-        filelist.close()
       }
 
+      // Move a file in a filelist which may not exist.
+      def maybeMoveFiles(filename: String) = try {
+        moveFiles(filename)
+      } catch {
+        case _ @(_: java.nio.file.NoSuchFileException | _: java.io.FileNotFoundException) =>
+      }
```

Comment:
It would be better to write this as "check if exists, then move if so" but whatever, very Pythonic if you 😉